### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by giuliopaci**
While reading scripts I found a few small errors.
The fixes are not related to each other, but they are trivial, so I hope a single pull request is ok.

### CODE REVIEW

File: tools/extras/install_irstlm.sh [hunk 1]
```diff
@@ -55,6 +55,7 @@ fi
   [ ! -z ${IRSTLM} ] && \
     echo >&2 "IRSTLM config is already in env.sh" && exit
 
+  wd=`pwd`
```
Comment (danpovey) [inline tools/extras/install_irstlm.sh]: why is this line added, isn't it overridden by the next line?
the '|| pwd' is about BSD compatibility.

File: tools/extras/install_srilm.sh [hunk 2]
```diff
@@ -24,7 +24,7 @@ tar -xvzf ../srilm.tgz
 # set the SRILM variable in the top-level Makefile to this directory.
 cp Makefile tmpf
 
-cat tmpf | awk -v pwd=`pwd` '/SRILM =/{printf("SRILM = %s\n", pwd); next;} {print;}' \
+cat tmpf | gawk -v pwd=`pwd` '/SRILM =/{printf("SRILM = %s\n", pwd); next;} {print;}' \
```
Comment (danpovey) [inline tools/extras/install_srilm.sh]: Kaldi requires awk to be installed anyway, so using awk is fine and actually recommended/preferred for Kaldi scripts...
we check for gawk because SRILM does not work without it.

File: tools/extras/install_irstlm.sh (hunk 1 above)
Comment (giuliopaci) [inline tools/extras/install_irstlm.sh]: This line is necessary because otherwise ```readlink -f $wd``` makes no sense and always fail.

File: tools/extras/install_irstlm.sh [hunk 3]
```diff
@@ -55,6 +55,7 @@ fi
...
   wd=`readlink -f $wd 2>/dev/null || pwd`
```
Comment (danpovey) [inline tools/extras/install_irstlm.sh]: oh I see.  you could probably replace $wd in this expression with $PWD, and remove the extra line.

File: tools/extras/install_srilm.sh (hunk 2 above)
Comment (giuliopaci) [inline tools/extras/install_srilm.sh]: Then I am going to fix another line in this script. Wait for the PR to be updated before merging.

Comment (giuliopaci) [inline tools/extras/install_srilm.sh]: Sorry, I missed your last line. Then I am going to drop this commit.

File: tools/extras/install_irstlm.sh (hunk 3 above)
Comment (giuliopaci) [inline tools/extras/install_irstlm.sh]: Reading http://www.unix.com/man-page/posix/1posix/pwd/ probably the desired behaviour is just one line:
```
wd=`pwd -P`
```

or, for maximum compatibility with already used systems:

```
wd=`pwd -P 2>/dev/null || pwd`
```

This will avoid the unportable ```readlink -f```, provide a portable solution and will be equivalent to or better than (from the performance point of view, that is completely irrelevant in this case) ```readlink -f $PWD``` in bash where ```pwd``` is a builtin.

Comment (danpovey) [inline tools/extras/install_irstlm.sh]: I'm OK with wd=`pwd -P`; it works on my mac.

**Comment by giuliopaci** [comment]
I just updated the pull request.

**Comment by danpovey** [comment]
Are you sure you pushed to your repo?  I dont see the w=`pwd -P`

**Comment by giuliopaci** [comment]
Indeed I pushed before noticing that I was still in the middle of a rebase, due to an error. Now it should be ok.

**Comment by danpovey** [comment]
Thanks-- merging.
